packages/website/data/sponsors.json

```diff
@@ -31,20 +39,12 @@
     "totalDonations": 120000,
     "website": "https://blog.coinbase.com/engineering-and-security/home"
   },
-  {
```

Comment:
This doesn't seem right!!!

Reply:
As in that the Airbnb entry shows as removed?

It’s because it leap frogged Coinbase above, there’s a new entry for it.

Please let me know if that’s not what you meant